app/new-home/-components/testimonials/template.hbs

```diff
@@ -0,0 +1,103 @@
+{{! template-lint-disable no-bare-strings }}
+<div data-test-testimonials-container local-class='container'>
+    <header>
+        <h1 data-test-testimonials-heading>
+            {{t 'osf-components.testimonials.heading'}}
+        </h1>
+    </header>
+    <Carousel as |container|>
+        <container.slide data-test-slide-1>
+            <div local-class='slide-content'>
+                <div>
+                    <span local-class='quote'>&quot;</span>
+                    <p>
+                        {{t 'osf-components.testimonials.slide_1.text'}}
+                    </p>
+                    <p local-class='description'>
+                        <span local-class='description-bold'>
+                            {{t 'osf-components.testimonials.slide_1.description.name'}}
+                        </span>
+                        <span>
+                            {{t 'osf-components.testimonials.slide_1.description.position'}}
+                        </span>
+                        <span local-class='description-bold'>
+                            <OsfLink
+                                data-analytics-name='noteworthy_search'
```

Comment:
`noteworthy_search`? It seems like each of these should be unique to tell us which one they clicked on? Maybe check with Sara?